Assets/Scripts/GrappleGunBehavior.cs

```diff
@@ -85,7 +85,7 @@ private bool GrappleBroken()
     {
         var grappleStart = lineRenderer.GetPosition(0);
         var grappleEnd = lineRenderer.GetPosition(1);
-        if (Physics.Raycast(grappleStart, grappleEnd - grappleStart, out var hit, Mathf.Infinity))
+        if (Physics.Raycast(grappleStart, grappleEnd - grappleStart, out var hit, Mathf.Infinity, grappleableStuff))
```

Comment:
what'd you change about grappleable stuff...? Wasn't it already working like this?

Reply:
I actually mapped a "grappleable" layer for asteroids and the exit portal (and soon heal pods), and also made it so that the grapple only breaks if the grapple point is covered by a grappleable object (e.g. it shouldn't break when you are grappling to an asteroid that's within the black hole pull collider just because the raycast will hit the pull collider before the grapple point)